tools/CMakeLists.txt

```diff
@@ -231,12 +231,14 @@ iree_cc_library(
     iree-prof-output
   HRDS
     "iree-prof-output.h"
+    "iree-prof-output-json.h"
```

Comment:
json seems too vague to represent chrome-tracing specifically. Might need a renaming.

Reply:
changed to iree-prof-output-chrome.